openrpc.json

```diff
@@ -369,6 +369,52 @@
           }
         }
       ],
+      "errors": [
+        {
+          "code": -32602,
+          "message": "Must specify address, symbol, and decimals."
+        },
+        {
+          "code": -32602,
+          "message": "Invalid symbol: not a string."
+        },
+        {
+          "code": -32602,
+          "message": "Invalid symbol '${symbol}': longer than 11 characters."
+        },
+        {
+          "code": -32602,
+          "message": "Invalid decimals '${decimals}': must be 0 <= 36."
+        },
+        {
+          "code": -32602,
+          "message": "Invalid address '${address}'."
+        },
```

Comment:
@BelfordZ @shane-t is there some way to indicate that these errors apply when param type = `ERC20` and the others apply when param type = `ERC721` or `ERC1155`

Reply:
There's no way to link errors with the param types, other than just mentioning it in the description

Reply:
@shanejonas Do you think it makes sense to describe this in the root description field?

Reply:
ping here @shanejonas ?

Reply:
I think its fine to have diff errors from diff params here, like decimals only applies to erc20 but we don't explicitly say that anywhere in the error.

Reply:
> but we don't explicitly say that anywhere in the error.

I can't tell, are you suggesting that we should say it explicitly in the error, or that its okay that we don't say it explicitly?

@shanejonas